src/main/java/frc/robot/commands/DriveStraightCommand.java

```diff
@@ -45,6 +45,19 @@ public DriveStraightCommand(NavXGyroSubsystem subsystem, DrivetrainSubsystem dri
     m_driveSpeed = 0.0;
   }
 
+  public DriveStraightCommand(NavXGyroSubsystem subsystem, DrivetrainSubsystem drivesubsystem, BlinkinSubsystem lightSubsystem, double leftSpeed, double rightSpeed, double throttle) {
+    m_subsystem = subsystem;
+    m_drivesubsystem = drivesubsystem;
+    m_ledSubsystem = lightSubsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem, drivesubsystem);
```

Comment:
Do we need `lightSubsystem` here too?
```suggestion
    addRequirements(subsystem, drivesubsystem, lightSubsystem);
```

Reply:
Nope

Reply:
ok, thanks. Why not?

Reply:
to clarify what I mean: since we use it here, shouldn't we add it to the requirements call.